bot/exts/halloween/hacktober-issue-finder.py

```diff
@@ -103,7 +103,7 @@ def format_embed(issue: Dict) -> discord.Embed:
         labels = [label["name"] for label in issue["labels"]]
 
         embed = discord.Embed(title=title)
-        embed.description = body
+        embed.description = body[:500] + '....' if len(body) > 500 else body
```

Comment:
I’m being very picky, but would it be possible to use only 3 dots here? That’s what we usually use.
```suggestion
        embed.description = body[:500] + '...' if len(body) > 500 else body
```

Reply:
You should just use `textwrap.shorten` instead.